browser/src/control/Control.PartsPreview.js

```diff
@@ -615,36 +615,17 @@ L.Control.PartsPreview = L.Control.extend({
 		}
 	},
 
-	_onScroll: function (e) {
-		setTimeout(L.bind(function (e) {
-			var scrollOffset = 0;
-			if (e) {
-				var prevScrollY = this._scrollY;
-				var rectangle = e.target.getBoundingClientRect();
-				this._scrollY = this._direction === 'x' ? -rectangle.left : -rectangle.top;
```

Comment:
if we remove this._scrollY then there are no uses of _scrollY left and we should remove the remaining initialization of this._scrollY = 0 earlier in this file.

Reply:
Thanks, I will remove it